roles/server_monitoring/templates/default.conf.j2

```
@@ -56,6 +74,14 @@ server {
         deny all;
         return 404;
 	}
+    {% if neutral_error_pages %}
+    error_page 400 401 403 404 500 502 503 504 /error.html;
+    location = /error.html {
+        root /var/www/html;
+        internal;
+        return 500 '<html><body><h1>Error Code: $status</h1><p>Something went wrong. Please try again later or contact your administrator.</p></body></html>';      
```

Wouldn't this mask the original error code by overwriting it with a 500